tpcx_bb/benchmark_runner.py

```diff
@@ -1,45 +1,93 @@
+import glob
+import re
 import os
 import gc
 import time
 
+
 os.environ["tpcxbb_benchmark_sweep_run"] = "True"
 
-qnums = [str(i).zfill(2) for i in range(1, 31)]
-n_repeats = 3
+N_REPEATS = 1
+
+
+def get_qnum_from_filename(name):
+    m = re.search("[0-9]{2}", name).group()
+    return m
+
+
+dask_qnums = [str(i).zfill(2) for i in range(1, 31)]
+
+# Not all queries are implemented with BSQL
+bsql_query_files = sorted(glob.glob("./queries/q*/t*_sql.py"))
+bsql_qnums = [get_qnum_from_filename(x.split("/")[-1]) for x in bsql_query_files]
 
 
 if __name__ == "__main__":
     from xbb_tools.cluster_startup import attach_to_cluster, import_query_libs
-    from xbb_tools.utils import run_dask_cudf_query, tpcxbb_argparser
+    from xbb_tools.utils import run_query, tpcxbb_argparser
     import importlib
 
     import_query_libs()
 
-    q_func_d = {
+    dask_queries = {
         qnum: importlib.import_module(f"queries.q{qnum}.tpcx_bb_query_{qnum}").main
-        for qnum in qnums
+        for qnum in dask_qnums
+    }
+
+    bsql_queries = {
+        qnum: importlib.import_module(f"queries.q{qnum}.tpcx_bb_query_{qnum}_sql").main
+        for qnum in bsql_qnums
     }
 
     cli_args = tpcxbb_argparser()
-    client = attach_to_cluster(cli_args)
+    client, bc = attach_to_cluster(cli_args, create_blazing_context=True)
```

Comment:
To make sure I understand this.. if a user wants to run a subset of queries that happen to be Dask only, should they set `create_blazing_context=False`?

I suppose it wouldn't really matter, as the context will be created once, and not in every query, correct?

Reply:
Yes, but this needs to be wrapped, as @ayushdg commented below. Otherwise we can't do a clean full RMM pool run with UCX and dask-only

Reply:
Resolved by https://github.com/rapidsai/tpcx-bb/pull/33/commits/f49574f11a479ec0e1ff7ae4f1ce0742cf9574cc